### DISCUSSION TITLE
Memes!!!

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/spring-framework#8] [Show and tell]
Feel free to share any memes here. No offence will be given or taken

**Comment by Mrafe** [comment]
![WhatsApp Image 2022-03-30 at 11 03 11 AM](https://user-images.githubusercontent.com/55877552/160877783-4a1970b2-e743-40b6-af67-0c5560b297cf.jpeg)

**Comment by Mrafe** [comment]
![WhatsApp Image 2022-03-30 at 2 24 24 PM](https://user-images.githubusercontent.com/55877552/160877931-20dd3edf-b785-43ed-be3d-011fb7633eda.jpeg)

**Comment by Mrafe** [comment]
https://user-images.githubusercontent.com/55877552/162179071-8e1a51f7-9e06-4bd3-b5b6-c1ac40b25c5c.mp4

**Reply by akash-coded** [reply]
![its-okay-you-can-do-it-memes-funny-pics-53062816](https://user-images.githubusercontent.com/60097560/162382540-38af27e2-b8b8-4a5b-82e6-22c1b60fce00.png)

**Comment by akash-coded** [comment]
![last_day](https://user-images.githubusercontent.com/60097560/162382615-ffd9c6e2-343c-4c40-a3af-067587f67dc5.jpg)

**Comment by aish955** [comment]
![WhatsApp Image 2022-03-25 at 10 42 13 AM](https://user-images.githubusercontent.com/70426431/162392401-1592126c-a27a-4520-bc0b-9d31790f6cb2.jpeg)

**Reply by akash-coded** [reply]
@aish955 
![this-is-fine-aish](https://user-images.githubusercontent.com/60097560/162509741-7bad761e-8893-46a3-aa94-c671fbffab14.jpg)

**Comment by Mrafe** [comment]
https://drive.google.com/file/d/1CHzdTi-sfiLTXqD0Yi1y2YcAO09cssnK/view?usp=sharing

**Comment by Ashwesh09** [comment]
![Screenshot_2022-04-07-23-25-55-69_1c337646f29875672b5a61192b9010f9](https://user-images.githubusercontent.com/66617894/162485482-c0ecdcbe-89e8-4837-bac1-d2f609e8024f.jpg)

**Comment by lalit-ky** [comment]
https://drive.google.com/file/d/1K-Pkicob_xYvYX0Cr2_7U9BYIYfOEsdl/view?usp=sharing
